File: Day11.rb
require_relative 'auth'

puts("[Please enter your ID] \n")
input_id = gets.chomp()
# module Auth
#   module_function()
#   def login(id)
#     members = ['mc11' , 'dt2002']
#     for member in members do
#           if member == id
#             return true
#           end
#       end
#       return false
#   end
# end
if Auth.login(input_id)
  puts('** ' + input_id + '!! Welcome to ITW Database hub.')
else
  puts('** Invalid ID **')
end

File: Day11.py


import Day9_2

print(('-' * 12) + ' Function : Login ' + ('-' * 12))
input_id = input("[Please enter your ID] \n")
if Day9_2.login(input_id):
    print("** G'day, " + input_id + "!! Welcome to ITW database hub")
else:
    print("** Invalid ID" + '\a')
